ffi/framebuffer_android.lua

```diff
@@ -149,34 +176,39 @@ end
 function framebuffer:refreshPartialImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
-        self:_updatePartial(partial_regal, 0, x, y, w, h)
+        -- in case of qualcomm it's actually partial_gc16
+        self:_updatePartial(partial_regal, delay_page, x, y, w, h)
     end
 end
 
 function framebuffer:refreshFlashPartialImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
-        self:_updatePartial(full_regal, 0, x, y, w, h)
+        self:_updatePartial(full_regal, delay_page, x, y, w, h)
     end
 end
 
 function framebuffer:refreshUIImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
-        self:_updatePartial(partial_regal, 0, x, y, w, h)
+        -- in case of qualcomm it's actually partial_gc16
+        self:_updatePartial(partial_regal, delay_ui, x, y, w, h)
     end
 end
 
 function framebuffer:refreshFlashUIImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
-        self:_updatePartial(full_regal, 0, x, y, w, h)
+        self:_updatePartial(full_regal, delay_ui, x, y, w, h)
     end
 end
 
 function framebuffer:refreshFastImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
+        -- qualcomm shouldn't have refresh here, since the update will "merge",
```

Comment:
like with "delay_ui" and "delay_page" I wouldn't mind a "delay_fast" set to 0 for obvious reasons :)